patches/components-permissions-permission_uma_util.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/permissions/permission_uma_util.h b/components/permissions/permission_uma_util.h
+index f2fe29c36726b73a92c8693466240e92340415c3..5b730ad861a80b9b315350e58bf47b6a4f8764aa 100644
+--- a/components/permissions/permission_uma_util.h
++++ b/components/permissions/permission_uma_util.h
+@@ -66,6 +66,7 @@ enum class RequestTypeForUma {
+   PERMISSION_WINDOW_PLACEMENT = 25,
+   PERMISSION_FONT_ACCESS = 26,
+   PERMISSION_IDLE_DETECTION = 27,
++  BRAVE_PERMISSION_REQUEST_TYPES_FOR_UMA
```

Comment:
do we need these? uma should be disabled

Reply:
We need one for Widevine for `GetUmaValueForRequestType` in `permission_uma_util.cc` because it maps `RequestType` to `RequestTypeForUma` in a switch statement and we get `enumeration value 'kWidevine' not handled in switch`. Though, I guess I could just map it to one of the existing values instead?

And the WALLET one can be removed, we don't even have that request type any more.

Reply:
Switched to using an existing RequestTypeForUma value instead of adding our own in 29da84e68ed6ed972762c23d0a3b7297fca1513d